apps/sync/js/fxsync-webcrypto/fxsyncwebcrypto.js

```diff
@@ -0,0 +1,279 @@
+'use strict';
```

Comment:
Let's name this folder "crypto" instead of "fxsync-webcrypto".

And add the MPL2 header, please.